ultralytics/data/explorer/explorer.py

```diff
@@ -466,3 +469,136 @@ def generate_report(self, result):
         TODO
         """
         pass
+
+    def count_labels(self, labels: list):
+        """
+        Count occurrences of given labels in dataset.
+
+        Args:
+            labels (list): List of labels to count their occurrences.
+
+        Returns:
+            filtered_df (pandas.DataFrame): Dataframe containing records containing given labels.
+            label_counts (dict): Dictionary containing labels with their count.
+
+        Example:
+            ```python
+            exp = Explorer()
+            exp.create_embeddings_table()
+            filtered_data, label_counts = exp.count_labels(labels)
+            ```
+        """
+        label_counts = {}
+        df = self.table.to_pandas()
+        filtered_df = df[df["labels"].apply(lambda l: bool(set(l) & set(labels)))]
+        flatten_labels = [l for ls in list(filtered_df.labels) for l in ls]
+        for label in labels:
+            if label in flatten_labels:
+                label_counts[label] = flatten_labels.count(label)
+            else:
+                label_counts[label] = 0
+        return filtered_df, label_counts
```

Comment:
Based on the function name, it should only return label_counts no? what is the use of filtered_df?

Reply:
Ohhh yes, Actually I thought to plot it to images but I'll change it to histogram that will resolve this